app/services/bookings/gitis/auth.rb

```diff
@@ -53,8 +64,15 @@ def auth_url
         Addressable::Template.new(AUTH_URL).expand(tenant_id: @tenant_id).to_s
       end
 
+      def faraday
+        Faraday.new do |f|
+          f.request(:retry, GET_RETRY_OPTIONS)
+          f.adapter(Faraday.default_adapter)
+        end
+      end
+
       def retrieve_token
-        response = Faraday.post(auth_url) do |request|
+        response = faraday.post(auth_url) do |request|
```

Comment:
Seems like we need to set the timeout on the request here, currently falling through to the default faraday timeout.

Reply:
Oops - good catch

Reply:
Ok that should now be fixed